sdk/core/Azure.Core/src/Serialization/AzureJsonSerializer.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Reflection;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+#pragma warning disable CS1591, SA1649, SA1402
+
+namespace Azure.Core.Serialization
+{
+    // For libraries that operate on opaque binary data (event hubs)
+    public abstract class AzureObjectSerializerFactory
```

Comment:
Do we really need these factories?

Reply:
Not sure if they would survive. My mental model was to have 1 serializer instance per type.